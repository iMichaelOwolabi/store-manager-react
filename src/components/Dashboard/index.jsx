import React, { Fragment } from 'react';
import AuthNav from '../AuthNav';

const Dashboard = () => (
  <Fragment>
    <AuthNav />
    <div className="container">
      <h1>Welcome to your dashboard</h1>
    </div>
  </Fragment>
);
export default Dashboard;
