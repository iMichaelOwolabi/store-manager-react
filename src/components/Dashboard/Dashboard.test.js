import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '.';

describe('<Narbar Component />', () => {
  it('should', () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
