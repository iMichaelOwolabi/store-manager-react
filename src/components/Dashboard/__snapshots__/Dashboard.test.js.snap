// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Narbar Component /> should 1`] = `
<Fragment>
  <AuthNav />
  <div
    className="container"
  >
    <h1>
      Welcome to your dashboard
    </h1>
  </div>
</Fragment>
`;
